docs/tutorials/getting-started.md

```diff
@@ -211,14 +211,44 @@ fig
 
 The `<py-repl>` tag creates a REPL component that is rendered to the page as a code editor, allowing you to write executable code inline.
 
-## Visual components
+## The py-config tag
 
-The following tags control visual attributes of your HTML page.
+Use the `<py-config>` tag to set and configure general metadata about your PyScript application in YAML format. If you are unfamiliar with YAML, consider reading [Red Hat's YAML for beginners](https://www.redhat.com/sysadmin/yaml-beginners) guide for more information.
+
+The `<py-config>` tag can be used as follows:
+
+```
+<py-config>
+  - autoclose_loader: false
+  - runtimes:
+    -
+      src: "https://cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js"
+      name: pyodide-0.20
+      lang: python
+</py-config>
+```
+
+The following values are supported by `<py-config>` (all values are optional):
+
+  * autoclose_loader (boolean): If false, PyScript will not close the loading splash screen when the startup operations finish.
+  * name (string): Name of the user application. This field can be any string and is to be used by the application author for their own customization purposes.
+  * version (string): Version of the user application. This field can be any string and is to be used by the application author for their own customization purposes. It is not related to the PyScript version.
+  * runtimes (List of Runtimes): List of runtime configurations. Each Runtime expects the following fields:
+    * src (string): URL to the runtime source.
+    * name (string), Optional: Name of the runtime. This field can be any string and is to be used by the application author for their own customization purposes.
+    * name (string), Optional: Programming language supported by the runtime. This field can by used by the application author to provide clarify. It currently has no implications on how PyScript behaves.
```

Comment:
@fpliger "name (string), Optional:" is repeated, should that second one be `language (string), Optional:` or something?

Reply:
Oh, sorry, typo. Yes, it's `lang`. Mind fixing? I'm afk right now.

Reply:
not at all! On it.